frontend/src/components/JsonSchemaForm.tsx

```diff
@@ -32,6 +32,14 @@ const renderers: Record<FieldType, (field: FieldDefinition) => JSX.Element> = {
   checkbox: (field: FieldDefinition) => {
     return (
       <div className="flex items-center mb-4">
+        <input
+          id={`${field.name}_hidden`}
+          name={field.name}
+          type="hidden"
+          checked={true}
```

Comment:
Why do you need the checked property and set it true, even though the value is "false"?
BTW I didn't test it, so if it works I trust you :)

Reply:
yeah I read that, OK no worries, it's fine to me!